### DISCUSSION TITLE
How use lua script get gen_vdp/0/m_vram memory data

### DISCUSSION THREAD

**Discussion by uname1able** [mamedev/discussions#131] [Q&A] [answered]
I  use   manager.machine.devices[":gen_vdp/0/m_vram"]  or  ["Sega 315-5313 Megadrive VDP:/gen_vdp/0/m_vram"] get nil , how implent the code to get gen_vdp/0/m_vram memory  by lua.

**Comment by cuavas** [comment, marked as answer]
The current way of doing it is unsupported, and will change in an incompatible way in the future.

```lua
vdp = manager.machine.devices['gen_vdp']
vram = emu.item(vdp.items['0/m_vram'])
```

Note that accessing the `items` property is very slow, so avoid doing it frequently.

**Reply by uname1able** [reply]
Thank you!!!
